### PR TITLE
fix(blocks): canvas flickering issue when resizing window

### PR DESCRIPTION
**Pull request by fundon**
Closes: [BS-185](https://linear.app/affine-design/issue/BS-185/connector-在调整窗口大小时会错位)

### CODE REVIEW

**Comment by fundon** [comment]
* **#7205** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7205?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by doouding — APPROVED** [review]

File: packages/blocks/src/surface-block/canvas-renderer/renderer.ts [hunk 1]
```diff
@@ -77,18 +77,25 @@ export class Renderer extends Viewport {
   }
 
   private _initViewport() {
+    let sizeUpdatedRafId: number | null = null;
+
     this.viewportUpdated.on(() => {
```
Comment (golok727) [inline packages/blocks/src/surface-block/canvas-renderer/renderer.ts:82]: Don't we need to add this to disposable?

Comment (golok727) [reply packages/blocks/src/surface-block/canvas-renderer/renderer.ts:82]: Ahh, we dispose it in the viewport😅

**Comment by golok727** [comment]
We are not disposing the Renderer._disposable in the `dispose` method

**Review by golok727 — APPROVED** [review]

**Pull request merged**